src/main/java/fr/xephi/authme/listener/PlayerListener.java

```diff
@@ -207,6 +215,11 @@ public void onPlayerJoin(PlayerJoinEvent event) {
         if (!PlayerListener19Spigot.isPlayerSpawnLocationEventCalled()) {
             teleportationService.teleportOnJoin(player);
         }
+
+        if (quickCommandsProtectionManager.shouldSaveLogin(player)) {
+            quickCommandsProtectionManager.setLogin(player.getName());
+        }
```

Comment:
I would personally merge both methods into something like `processLogin(player)` (or saveLogin or whatever) and the cmd protection manager itself takes care of checking the permission node.

Reply:
Perfect, editing ;)